src/test/java/ru/training/at/hw2/BaseJdiTestingIndexPageTest.java

```diff
@@ -0,0 +1,72 @@
+package ru.training.at.hw2;
+
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw2.data.JdiTestingIndexPageData;
+
+public class BaseJdiTestingIndexPageTest extends SeleniumTestsSetup {
+    protected WebDriver driver;
+
+    protected SoftAssert assertAccumulator;
```

Comment:
SoftAssert is part of a test method, not a test class. Because some tests may need it, some may not.
Please put it to the appropriate place